samples/openthread/coap_client/README.rst

```diff
@@ -124,6 +132,7 @@ To activate the optional extensions supported by this sample, modify :makevar:`O
 
 * For the Minimal Thread Device variant, set :file:`overlay-mtd.conf`.
 * For the Multiprotocol Bluetooth LE extension, set :file:`overlay-multiprotocol_ble.conf`.
+* For the Optimized Multiprotocol Bluetooth LE extension, set :file:`"overlay-multiprotocol_ble.conf overlay-optimization_multiprotocol_ble.conf"`.
```

Comment:
Maybe we should somehow indicate that optimization_multiprotocol_ble is not to be used when sed variant is used.

Reply:
Why is that? I was sure that it is possible with the MTD/SED variant.

Reply:
The system workqueue stack size is too low for switching sed->med mode and tha app crashes.

Reply:
I've adjusted the size of system workqueue for MTD variant.  This should be enough.